src/portal-components/DetailsPage.tsx

```diff
@@ -448,7 +448,8 @@ export default class DetailsPage extends React.Component<
           value = name
 
           // For explorer 2.0, construct an object to contain the locked facet name and facet value
-          lockedFacet.facet = columnName
+          // Make sure locked facet name is lowercase 
+          lockedFacet.facet = columnName.toLowerCase()
```

Comment:
Not sure if this is always true, but understand why this might be necessary in the AD Knowledge portal ("Study" vs the file view "study" column).  Will work in PsychEncode today as well ("study" is already lowercase across tables).

Reply:
Yes, the lowercase function is added because AD Knowledge portal uses "Study" so it doesn't match the file view "study" column when comparing the 2 strings, therefore the locked facet was not working.